en/creating-artboard-adobexd.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Creating An Artboard in Adobe XD
+_description: How to Adobe XD Artboard for a registration screen using Indigo Design Libraries.
+_keywords: Design Systems, Design Systems UX, UI kit, Adobe XD, Ignite UI for Angular, Adobe XD to Angular, Angular, Angular Design System, Export code from Adobe XD, Design Kits for Angular, Adobe XD HTML, Adobe XD to HTML, Adobe XD UI kits
+---
+
+# Creating Your First Artboard in Adobe XD
+
+At the end of this topic you'll have an Adobe XD Artboard for a registration screen using the **Indigo.Design System**. If you use Sketch for design then jump to [this topic](creating-artboard-sketch.md).
```

Comment:
It would be great if you remind briefly viewers what are the benefits of using the design system principles.